metadata-ingestion/setup.py

```diff
@@ -651,6 +651,7 @@
         "redshift = datahub.ingestion.source.redshift.redshift:RedshiftSource",
         "slack = datahub.ingestion.source.slack.slack:SlackSource",
         "snowflake = datahub.ingestion.source.snowflake.snowflake_v2:SnowflakeV2Source",
+        "snowflake-summary = datahub.ingestion.source.snowflake.snowflake_summary:SnowflakeSummarySource",
```

Comment:
I didn't see the definition of dependencies for this source. Shouldn't it have `snowflake_common` as a dependency?

Reply:
resolved in a follow up PR